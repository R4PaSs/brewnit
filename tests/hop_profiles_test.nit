import dummy_db

with ctx = new DBContext do
	var res = ctx.hop_profile_worker.fetch_multiple(ctx, "* FROM hop_profiles")
	for i in res do print i
end
