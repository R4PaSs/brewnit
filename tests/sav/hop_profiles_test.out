Hop: Hallertau Mittlefruer, AA: 4.11, quantity: 40.0 g, use: Boil, time: 3600.0 s
Hop: Fuggles, AA: 5.5, quantity: 28.35 g, use: Boil, time: 3600.0 s
Hop: Simcoe, AA: 15.4, quantity: 50.0 g, use: Boil, time: 3600.0 s
